### ISSUE TITLE
cluster: Reference result interval has lower bound greater than upper bound

### ISSUE THREAD

**Issue by ahartmanns** [ahartmanns/qcomp#109]
For `cluster`, the reference result for parameters 64-2000-20 and property `below_min` is an interval whose lower bound is greater than its upper bound:

```
"property": "below_min",
"value": {
	"lower": 0.00421944387,
	"upper": 0.00421944367
},
"note": "Storm"
```

**Comment by ahartmanns** [comment]
@tquatmann, is this something you could take a quick look at? Is it just a mistake in the JSON file, or does the empty interval actually come from Storm?

**Comment by tquatmann** [comment]
I confirm that this must be a mistake in the JSON file. Sorry for that!

**Comment by ahartmanns** [comment]
So the JSON should say
```
"property": "below_min",
"value": {
	"lower": 0.00421944367,
	"upper": 0.00421944387
},
"note": "Storm"
```
i.e. we just exchange the lower and upper bound?

**Comment by ahartmanns** [comment]
Perfect, thanks. Fixed in 78117fdb386b3476baeece37fd049b3117d217b1.
